snap/snapcraft.yaml

```diff
@@ -15,6 +15,7 @@ apps:
   wifi-connect: 
     command: cmd 
     plugs: [network, network-control, network-bind, network-manager, control]
+    aliases: [wifi-connect_app]
```

Comment:
yeap. You're right. They still are available but will be deprecated soon. I've removed that test